### DISCUSSION TITLE
How to hide some of the drawing tools

### DISCUSSION THREAD

**Discussion by till90** [opengeos/leafmap#253] [Q&A] [answered]
Hi, 
is it possible to hide for example the polygon drawing tool in the toolbar?

**Comment by giswqs** [comment, marked as answer]
You can hide the built-in draw control (e.g., `m = leafmap.Map(draw_control=False)`) and add your own custom draw control. See 

https://ipyleaflet.readthedocs.io/en/latest/controls/draw_control.html
